lib/chef-dk/skeletons/code_generator/metadata.rb

```diff
@@ -5,4 +5,3 @@
 description      'Generates Chef code for Chef DK'
 long_description 'Generates Chef code for Chef DK'
 version          '0.1.0'
```

Comment:
I dont know if that would add a lot of value. We don't upload it to a Chef Server, and its already versioned via the ChefDK itself.